### ISSUE TITLE
Unable to create multiple macro in project scope and tenant

### ISSUE THREAD

**Issue by alishah730** [spectrocloud/terraform-spectrocloud-modules#59]
Unable to create multiple macro with Project scope getting below error 

```
module.SpectroMacro.spectrocloud_macro.macro["macro1-simyam%Default"]: Creating...
module.SpectroMacro.spectrocloud_macro.macro["macro2-simyam%Default"]: Creating...
╷
│ Error:  &{Code:ResourceNotFound Details:<nil> Message:Macro '631e303ae3a42544f34530b5' is not found in the project aa-macro-tf Ref:0f7ea0fc704f27b9650724d2cca1a1f2}
│ 
│   with module.SpectroMacro.spectrocloud_macro.macro["macro1-simyam%Default"],
│   on .terraform/modules/SpectroMacro/spectro-macro.tf line 18, in resource "spectrocloud_macro" "macro":
│   18: resource "spectrocloud_macro" "macro" {
│ 
╵
╷
│ Error:  &{Code:ResourceNotFound Details:<nil> Message:Macro '631e303ae3a42544f34530b5' is not found in the project aa-macro-tf Ref:0f7ea0fc704f27b9650724d2cca1a1f2}
│ 
│   with module.SpectroMacro.spectrocloud_macro.macro["macro2-simyam%Default"],
│   on .terraform/modules/SpectroMacro/spectro-macro.tf line 18, in resource "spectrocloud_macro" "macro":
│   18: resource "spectrocloud_macro" "macro" {
```

**Comment by nikchern** [comment]
@alishah730 thank you for reporting an issue. An internal bug is created based on the error message you provided. Please stay tuned for the fix.

**Comment by nikchern** [comment]
The issue was actually fixed a while ago with 0.5.4 release. Hence closing the ticket.
